starters/features/trpc-prisma/src/routes/trpc-prisma/index.tsx

```diff
@@ -0,0 +1,29 @@
+import { component$, useClientEffect$, useStore } from '@builder.io/qwik';
+import { Framework } from '@prisma/client';
+import { trpc } from '~/client/trpc';
+
+export default component$(() => {
+	const store = useStore({ items: [] as Framework[] });
+
+	useClientEffect$(async ({ cleanup }) => {
```

Comment:
doesn't this mean it will not fill the data in SSR?

Reply:
Yep, I will update the starter to be SSR compliant

Reply:
I added tRPC server side example